presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryMetadata.java

```diff
@@ -25,6 +25,7 @@
 {
     private final String queryId;
     private final Optional<String> transactionId;
+    private final Optional<String> tracingId;
```

Comment:
Can we explain a bit more why we need tracingId in QueryMetadata?

Reply:
This draft PR can explain this. https://github.com/facebookexternal/presto-facebook/pull/1751
Basically we want to export trace id to lookup page so need that trace id to be present in listener. I think QueryMetadata is a good place to have it.